b9lab-content/2-what-is-cosmos/03-cosmos-ecosystem.md

```diff
@@ -0,0 +1,177 @@
+
+The ready-built modules of the Cosmos SDK can be easily used - It is as simple as importing modules into the application. Additionally, developers can create their own modules to introduce specific functionalities. It is expected that, with the growth of the ecosystem, the number of modules will grow, thereby facilitating the development of more complex applications.
+
+*Building on modular components, many of those you did not write yourself? Although it sounds like an increased potential for attacks, and faulty or malicious nodes operating undetected? One does not have to worry.*
+
+The Cosmos SDK is built on a foundation of the [object-capability model](https://docs.cosmos.network/master/core/ocap.html). It not only favors modularity, it also encapsulates code implementation. Thus, capabilities constrain security boundaries between modules.
+
+The default consensus mechanism available to build with the SDK is [Tendermint Core](https://docs.tendermint.com/master/).
+
+## The Inter-Blockchain Communication Protocol
+
+The [Inter-Blockchain Communication (IBC) protocol](https://ibcprotocol.org/) is the basis for **interoperability** in Cosmos. It leverages Tendermint's instant finality to allow for the transfer of value (i.e. tokens) and data communication between heterogenous chains. Blockchains with different applications and architecture specifications become interoperable whether or not they share a validator set.
+
+Without IBC, the interoperability of heterogenous chains is difficult to achieve because they may implement the consensus, networking, and application layers in various ways. As soon as a blockchain is compatible with IBC, it becomes interoperable with others. IBC has few requirements to make a chain compatible: Mainly, the consensus layer must have fast and absolute finality.
+
+*With these chain-to-chain connections now possible, how can we create a network of blockchains?*
+
+The first naive idea would be to connect each blockchain with the other chains through direct IBC connections. Unfortunately, this would quickly lead to a very large, expensive, and ultimately unmanageable number of connections in the network. In fact, the number of connections in the network would increase quadratically with the number of blockchains.
+
+A second idea, one that **reduces the number of connections necessary** back down to the number of chains, would be to introduce an intermediate entity that facilitates the transfer between chains, in a hub and spoke model. Unfortunately, this would bring back unwelcome centralization.
+
+*How could we uphold decentralization then?*
+
+First, by keeping the capability of chain-to-chain connections made possible by **IBC**. Second, by creating a flexible multi-hub and spoke model whereby a spoke gets automatic access to all the other spokes of the hub it connected to. This is the road **Cosmos** took.
+
+Cosmos implements a **modular architecture with two blockchain classes**: **hubs** and **zones**.
```

Comment:
Should be more clear there that Hubs are just Zones with a lot of IBC connections. Some folks think the Hub/Zone terminology is confusing and should be abandoned all together. But it should be mentioned that there's no enforced topology of IBC chains, they can connect to whoever they want, but the expectation is that a hub and spoke model will emerge as some chains come to specialize in providing IBC connectivity to many others